scripts/run_benches_for_pallets.sh

```diff
@@ -19,17 +19,19 @@ pallets=(
 	pallet-web3-names
 	parachain-staking
 	public-credentials
+	pallet-asset-switch
 )
 
 // Add Peregrine-only pallets here!
 if [ "$runtime" = "peregrine" ]; then
   pallets+=(
-	pallet-asset-switch
-  )
+		pallet-bonded-coins
```

Comment:
The codebase still does not contain any runtime code, so this change should be introduced in that PR, not in here.

Reply:
Droped in: [8739367](https://github.com/KILTprotocol/kilt-node/pull/789/commits/87393679e7015a3bea258bf75433596e63e4e7d9)